Movecraft/src/main/java/net/countercraft/movecraft/craft/CruiseOnPilotSubCraft.java

```diff
@@ -23,4 +23,5 @@ public CruiseOnPilotSubCraft(@NotNull CraftType type, @NotNull World world, @Not
     public void setParent(@NotNull Craft parent) {
         this.parent = parent;
     }
+
```

Comment:
I was going to clean up this diff myself, but it looks like "allow edits from maintainers" wasn't selected when this PR was opened.

Reply:
I'll see what i can do

Reply:
CraftDataTagContainer being at craft level is cause the set and get methods are implemented using default methods in the interface, which requires access to the container in a way. And since interfaces can't have fields we need some method for it and the craft needs to implement it to give access to the field.